### ISSUE TITLE
Re: XBOFS auto minimizes but vanishes

### ISSUE THREAD

**Issue by hahahi123** [OOPMan/XBOFS.win#50]
hi oopmanza.  been having an issue with my xbofs app where it does function but i cant change the bindings when the app window suddenly vanishes & runs in the background.  the icon that used to appear on the taskbar doesn't show up anymore either. i can still use my stick but unfortunately i cant change bindings back to default.  it is not possible when the app is behaving this way. i tried using the faq to uninstall & reinstall but the same issue still happens.  is there a way to reset the xbofs cfg?  Thank you

**Comment by OOPMan** [comment]
Yes, you can reset the config. It is stored in Windows Registry:

0. Use Task Manager (CTRL+SHIFT+Esc) to kill the XBOFS.win process
1. Hit WIN+R
2. Enter `regedit` and press enter
3. Yes to elevate priviledges
4. Navigate to `HKEY_CURRENT_USER\SOFTWARE\OOPMan\XBOFS.win`
5. Change `minimizeToTray` to `false`
6. Change `startMinimized` to `false`
7. Exit Regedit
8. Start XBOFS.win

**Comment by hahahi123** [comment]
> Yes, you can reset the config. It is stored in Windows Registry:
> 
> 0. Use Task Manager (CTRL+SHIFT+Esc) to kill the XBOFS.win process
> 1. Hit WIN+R
> 2. Enter `regedit` and press enter
> 3. Yes to elevate priviledges
> 4. Navigate to `HKEY_CURRENT_USER\SOFTWARE\OOPMan\XBOFS.win`
> 5. Change `minimizeToTray` to `false`
> 6. Change `startMinimized` to `false`
> 7. Exit Regedit
> 8. Start XBOFS.win

wow. thanks. i'll try it out. its really great to see you again.  used to talk to you on reddit. hope all is well.

**Comment by OOPMan** [comment]
> > Yes, you can reset the config. It is stored in Windows Registry:
> > 
> > 0. Use Task Manager (CTRL+SHIFT+Esc) to kill the XBOFS.win process
> > 1. Hit WIN+R
> > 2. Enter `regedit` and press enter
> > 3. Yes to elevate priviledges
> > 4. Navigate to `HKEY_CURRENT_USER\SOFTWARE\OOPMan\XBOFS.win`
> > 5. Change `minimizeToTray` to `false`
> > 6. Change `startMinimized` to `false`
> > 7. Exit Regedit
> > 8. Start XBOFS.win
> 
> wow. thanks. i'll try it out. its really great to see you again. used to talk to you on reddit. hope all is well.

Yah, I quit reddit when they killed third party clients. I sometimes read stuff there but not going to login or post again.
